schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals(
+    id INTEGER,
```

Comment:
- [OPTIONAL] It would be nice to use the `ID` column as the **_primary key_**. The **_primary key_** ensures unique row identification. This results in faster sorting, searching, and querying operations. 👍 😎